Gems/Atom/RHI/Code/Include/Atom/RHI/MultiDeviceBuffer.h

```diff
@@ -63,17 +66,18 @@ namespace AZ::RHI
     };
 
     //! A MultiDeviceBufferView is a light-weight representation of a view onto a multi-device buffer.
-    //! It holds a raw pointer to a multi-device buffer as well as a BufferViewDescriptor object.
-    //! Using both, single-device BufferViews can be retrieved.
-    class MultiDeviceBufferView : public Object
+    //! It holds an RHI::Ptr to a multi-device buffer as well as a BufferViewDescriptor
```

Comment:
It is still holding a raw pointer to MultiDeviceBuffer below except it is now also caching all the RHI::Ptr to BufferView. We should add a comment about that explaining why it is doing that.